tests/scripts/analyze_outcomes.py

```diff
@@ -87,19 +87,19 @@ def analyze_coverage(results: Results, outcomes: Outcomes,
     """Check that all available test cases are executed at least once."""
     available = check_test_cases.collect_available_test_cases()
     for suite_case in available:
-        hits = 0
+        hit = False
         for comp_outcomes in outcomes.values():
             if suite_case in comp_outcomes.successes or \
                suite_case in comp_outcomes.failures:
-                hits += 1
+                hit = True
                 break
 
-        if hits == 0 and suite_case not in allow_list:
+        if hit == 0 and suite_case not in allow_list:
```

Comment:
Now that `hit` is a boolean, I'd suggest `if not hit` here, and `elif hit and ...` below.